docs/openapi/api-internal-b2b-pa-older-versions.yaml

```diff
@@ -1355,6 +1596,8 @@ components:
       $ref: './remote-refs.yaml#/components/schemas/TimelineElementV20'
     TimelineElementV23:
       $ref: './remote-refs.yaml#/components/schemas/TimelineElementV23'
+    TimelineElementV24:
+      $ref: './remote-refs.yaml#/components/schemas/TimelineElementV24'
```

Comment:
@simonerondinella é aumentata anche la versione della Timeline

Reply:
Nono non è stata aggiornata la versione della timeline (anche se nella feature di annullamento verrà poi fatto), in questo caso ho riportato semplicemente l'ultima versione della timeline nelle older version perchè è stata aggiornata la FullSentNotificationV24 in FullSentNotificationV25 e questa ha la timeline associata, riportando l'ogetto FullSentNotificationV24 nelle older version ho riportato anche il component della timelineV24